.gitignore

```diff
@@ -1,12 +1,8 @@
 .idea/
-jdbc/SQL-Scripts/
-twitter/twitter.iml
-twitter/target/
-jdbc/target/
-grep/target/
-jdbc/.idea/
-jdbc/target
-twitter/.idea/
-jdbc/jdbc.iml
-grep/grep.iml
+core_java/jdbc/SQL-Scripts/
+*.iml
+target/
```

Comment:
https://www.notion.so/jarviscanada/Git-Github-GitFlow-Guidelines-4254192fe01e41bbbb2cce3f85185cda#6e025aba746e44248df8c0ceffdf493f

Reply:
remember to update the main README file